.github/workflows/pr.yml

```diff
@@ -11,6 +11,40 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  build-binaries:
+    name: Build binaries
+    runs-on: ubuntu-latest
+    env:
+      ATHENA_DIR: "${{ github.workspace }}/.athena"
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Install Athena toolchain
+        run: |
+          curl -L https://install.athenavm.org | bash
+          $ATHENA_DIR/bin/athup --path $GITHUB_WORKSPACE
+          $ATHENA_DIR/bin/cargo-athena athena --version
+          echo "$ATHENA_DIR/bin" >> $GITHUB_PATH
+
+      - name: Install Athena CLI
+        run: |
+          cd cli
+          cargo install --force --locked --path .
+          cargo athena install-toolchain
```

Comment:
Pretty sure the `athup` command above installs the toolchain. I think you don't need both of these steps.

Reply:
I think it doesnt. cargo install is required for cargo athena to be available.

Reply:
don't you love that friendly, clear, helpful error message? :)

Reply:
If you just run `athup` in default mode, it both sets up `~/.cargo/bin/cargo-athena` _and_ sets up the toolchain. If you run it with `--path` as we do here, it does the former (with a link to the local build) but not the latter. (Do we want to change this?) So, you need to run `cargo athena install-toolchain` but you do _not_ need to run `cargo install --force --locked --path cli` -- I just double checked this. If `cargo athena` isn't working after the athup command, it must be a PATH issue.

Reply:
Fixed by removing the `athup` step. Now it is installing `cli` to have `cargo athena` available (needed for `make -C tests/examples`) and the toolchain (to actually build the binaries).